### PR TITLE
update newView, newsViewController, and newsViewCell

### PR DESCRIPTION
**Pull request by MattHuie**
Updated NewsView, NewsViewController, and NewsViewCell with new constraints and object optimization.

Delegate and DelegateFlowLayout methods all updated in NewsViewController.

### CODE REVIEW

**Comment by MattHuie** [comment]
@vhart Hey can you check this out when you can

**Comment by vhart** [comment]
@MattHuie could you add a UI screenshot? Conventionally, anytime UI is changed it's recommended to add a before and after image, or at least an after image. Helps readers understand the context of the changes.

File: StellarApp/Controllers/NewsViewController.swift [hunk 1]
```diff
@@ -80,14 +107,23 @@ extension NewsViewController: UICollectionViewDelegate, UICollectionViewDelegate
         present(safariVC, animated: true, completion: nil)
     }
     func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, sizeForItemAt indexPath: IndexPath) -> CGSize {
-        return CGSize(width: view.bounds.width - 32, height: 300)
+        let screenWidth = UIScreen.main.bounds.width
+        let screenHeight = UIScreen.main.bounds.height
+        let width = (screenWidth - (self.cellSpacing * self.numberOfSpaces)) / self.numberOfCells
+        let height = (screenHeight / screenWidth) * (width / 2.2)
```
Comment (vhart) [inline StellarApp/Controllers/NewsViewController.swift:113]: interesting choice. aspect ration * some base. Why `2.2`?

File: StellarApp/Controllers/View/NewsCollectionViewCell.swift [hunk 2]
```diff
@@ -12,64 +12,83 @@ class NewsCollectionViewCell: UICollectionViewCell {
...
-        
+        commonInit()
     }
     
     required init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
     
+    private func commonInit() {
+       setupConstraints()
```
Comment (vhart) [inline StellarApp/Controllers/View/NewsCollectionViewCell.swift:52]: spacing is a little off

**Review by vhart — APPROVED** [review]
LGTM

File: StellarApp/Controllers/NewsViewController.swift (hunk 1 above)
Comment (MattHuie) [reply StellarApp/Controllers/NewsViewController.swift:113]: It was just some arbitrary number , we can definitely change it later.

File: StellarApp/Controllers/View/NewsCollectionViewCell.swift (hunk 2 above)
Comment (MattHuie) [reply StellarApp/Controllers/View/NewsCollectionViewCell.swift:52]: Oh ill take a look at it

**Pull request merged**